public/app/features/serviceaccounts/ServiceAccountTokensTable.tsx

```diff
@@ -0,0 +1,73 @@
+import React, { FC } from 'react';
+import { DeleteButton, Icon, IconName, Tooltip, useTheme2 } from '@grafana/ui';
+import { dateTimeFormat, GrafanaTheme2, TimeZone } from '@grafana/data';
+
+import { ApiKey } from '../../types';
+import { css } from '@emotion/css';
+
+interface Props {
+  apiKeys: ApiKey[];
```

Comment:
I think since we use name `tokens`, we should do it consistently.
```suggestion
  tokens: ApiKey[];
```